scripts/biospecimen_count.Rmd

```diff
@@ -73,8 +73,8 @@ Read in tables. The file `histologies.tsv` contains the cancer assignment for ea
 read_tsv('../OpenPedCan-analysis/data/histologies.tsv') -> histologies
```

Comment:
It is preferred to add your variables at the beginning of your line - I don't think this needs to be updated now, but can you add a ticket to update the code a bit? I also suggest that you print the tables in this RMD - the purpose of the notebook is so people can see and explore the data a bit interactively/see your plots as they are generated if not final and without printing these, they are not "exploreable", so you may as well have an R script instead. 

Separately and also not needed for this but an update we should make in the future is that I think we should do counts by tumor instead of specimen and/or counts for both - people often ask "how many tumors in the cohort" vs samples. I didn't dig much into the code, but it is also possible that your numbers here are indeed number of tumors since you are using independent specimens, so if that is the case, can you update to use the word "tumors" instead of "biospecimens"? It will make the numbers/table clearer to the reader.

Reply:
Updates discussed in this comment as well as the other comments in this PR have been added to https://github.com/PediatricOpenTargets/ticket-tracker/issues/462